src/nwsapi.js

```diff
@@ -534,7 +534,7 @@
         // contentType not in IE <= 11
         'contentType' in doc ?
           doc.contentType.indexOf('/html') > 0 :
-          doc.createElement('DiV').nodeName == 'DIV';
+          doc.createElement('DiV').localName == 'DIV';
```

Comment:
`localName` reflects the element name without **ASCII** upper‑casing, so it will be `"DiV"` and not `"DIV"`.

Reply:
@ExE-Boss
Thank you for the help following with us.
In both Firefox and Chrome (latest versions):
document.createElement('DiV').localName == 'div'
instead of 'DiV'. Any thought ?

If this is true then `localName' is the lower-casing version of the tag name.

Recall that this differentiation was introduced to support mixed namespaces in the same document (SGV mostly).
Do you believe we are at risk of introducing regressions related to that multi-namespaces support with these changes ?

Reply:
Seems like we could just switch to 
```suggestion
          doc.createElement('DiV').localName == 'div';
```

Reply:
The lowercasing of `localName` only happens in HTML documents:

- <https://dom.spec.whatwg.org/#dom-document-createelement>

Reply:
@ExE-Boss 
Ok, by reading that I understand that the functionality of `localName` depends on the document type.